test/cypress/integration/admin/admin-helpers.cy.js

```diff
@@ -203,10 +203,15 @@ export class AdminPage {
   }
 
   clickBlock(name) {
```

Comment:
Should we remove this `name` that the frontend action flagged?

Reply:
Thanks! Nice catch. Let me fix.